src/Arcus/AbstractIPAddressRange.cs

```diff
@@ -395,5 +395,37 @@ public bool Touches(IIPAddressRange that)
         #endregion // end: Ovelap and Touches
 
         #endregion // end: Set Operations
+
+        #region Contains Any/All Public/Private Addresses
+
+        /// <inheritdoc/>
+        public bool ContainsAnyPrivateAddresses()
+        {
+            return SubnetUtilities.PrivateIPAddressRangesList.Any(subnet => subnet.Contains(this.Head)
```

Comment:
This is the primary reason why I think the naming of `SubnetUtilities` is odd, given its usage here.

Reply:
Do you have a proposal for an alternate solution?
We define constants / read only values in the `IPAddressUtils` as well. I'm not really sure where else they'd belong.

Reply:
Just referring to the naming, but your reasoning is sound. Going to resolve.